packages/commonwealth/client/scripts/App.tsx

```diff
@@ -41,14 +37,8 @@ const App = () => {
                     <OnBoardingWrapperForMobile>
                       <ReactNativeBridgeUser />
                       <ReactNativeLogForwarder />
+                      <ReactNativeScrollToTopListener />
```

Comment:
how about moving all those components to one main react native parent component?

Reply:
Next iteration I'm going to have them re-use functions but they are more readable if they're isolated. 

Smashing them together would make them harder to understand.

Plus there's no performance advantage or anything else other than a few names.